modules/server_processes_manager/php/NDB_Menu_Filter_server_processes_manager.class.inc

```diff
@@ -0,0 +1,130 @@
+<?php
+/**
+ * Page used to view/manage the processes launched asynchronously by LORIS users.
+ *
+ * PHP Version 5
+ *
+ * @category ServerProcessLauncher
+ * @package  Loris
+ * @author   Nicolas Brossard <nicolasbrossard.mni@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+
+require_once "ServerProcessesMonitor.class.inc";
+
+/**
+ * Builds a new filter to view/manage processes launched asynchronously by LORIS
+ * users.
+ *
+ * PHP Version 5
+ *
+ * @category ServerProcessLauncher
+ * @package  Loris
+ * @author   Nicolas Brossard <nicolasbrossard.mni@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://github.com/aces/Loris
+ */
+class NDB_Menu_Filter_Server_Processes_Manager extends NDB_Menu_Filter
+{
+    /**
+     * Determines whether the current user has access to the page.
+     *
+     * @note   overloaded function
+     * @return bool true if the user has access, false otherwise
+     * @access private
+     */
+    function _hasAccess()
+    {
+        $user =& User::singleton();
```

Comment:
I think this will cause a warning about only variables being able to be assigned by references, and will stop working in PHP7. It should be fine to just get rid of the ampersand, since the singleton function is already defined to return a reference.